posthog/hogql_queries/insights/trends/test/test_trends.py

```diff
@@ -4745,7 +4753,7 @@ def test_breakdown_filtering(self):
                 self.team,
             )
 
-        self.assertEqual(response[0]["label"], "sign up - none")
+        self.assertEqual(response[0]["label"], "sign up - $$_posthog_breakdown_other_$$")
         self.assertEqual(response[1]["label"], "sign up - value")
         self.assertEqual(response[2]["label"], "sign up - other_value")
         self.assertEqual(response[3]["label"], "no events - none")
```

Comment:
This has to stay as `none`. Because there's no events, there are no breakdown values, and therefore we don't know the type of the values, therefore we dont know which version of the "other" value to use. Using the wrong one causes a clickhouse error with mismatched types with no supertype